c/go/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
This file should not be in the top-level directory of the `c/go` subproject but rather next to the example's source file.

For that to work, we need to tweak the `Makefile` along these lines:

```
diff --git a/c/go/Makefile b/c/go/Makefile
index 1b7f402..25d6a84 100644
--- a/c/go/Makefile
+++ b/c/go/Makefile
@@ -30,9 +30,12 @@ test: gobuild
 
 run-examples: build
        for example in $$(ls examples/); do \
-               $(MAKE) -C examples/$$example || exit 1; \
-               examples/$$example/$$example || exit 1; \
-               $(MAKE) -C examples/$$example clean || exit 1; \
+               ( \
+                       cd examples/$$example || exit 1; \
+                       $(MAKE) || exit 1; \
+                       ./$$example || exit 1; \
+                       $(MAKE) clean || exit 1; \
+               ) || exit 1; \
        done
 
 fmt:
```

I'll do it in a separate patch soon.

Reply:
> I'll do it in a separate patch soon.

Done: https://github.com/virtblocks/virtblocks/commit/dfdb5db869834f7a86dbb912d32c26058fd29188.